PayPal/src/main/java/com/braintreepayments/api/PayPalLauncher.kt

```diff
@@ -0,0 +1,94 @@
+package com.braintreepayments.api
+
+import android.content.Intent
+import androidx.activity.ComponentActivity
+
+/**
+ * Responsible for launching PayPal user authentication in a web browser
+ */
+class PayPalLauncher internal constructor(private val browserSwitchClient: BrowserSwitchClient) {
+    /**
+     * Used to launch the PayPal flow in a web browser and deliver results to your Activity
+     */
+    constructor() : this(BrowserSwitchClient())
```

Comment:
Could we instead make this a default value in the main constructor? 

```kotlin
class PayPalLauncher internal constructor(
    private val browserSwitchClient: BrowserSwitchClient = BrowserSwitchClient()
)
```

edit: actually is this to only expose the empty constructor to the merchant?

Reply:
Yes we don't want to expose `BrowserSwitchClient` to merchants, but we want them to have an empty constructor.